### PR TITLE
Add Ethical Considerations

### PR DESCRIPTION
**Pull request by anssiko**
***
<a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/pull/227.html" title="Last updated on Nov 11, 2021, 2:56 PM UTC (2b16e72)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/227/5ddb8ad...2b16e72.html" title="Last updated on Nov 11, 2021, 2:56 PM UTC (2b16e72)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -391,6 +391,10 @@ Note: The group is <a href="https://github.com/webmachinelearning/webnn/issues/8
 
 Implementers of this API are expected to be familiar with the <a href="https://gpuweb.github.io/gpuweb/#security-privacy">WebGPU Privacy Considerations</a>.
 
+Ethical Considerations {#ethics}
+===================================
+
+The Working Group has started documenting ethical issues associated with using Machine Learning on the Web, to help identify what mitigations its normative specifications should take into account. This work currently happens in a dedicated <a href="https://github.com/webmachinelearning/ethical-webmachinelearning">GitHub repository</a>.
```
Comment (jyasskin) [inline index.bs:397]: At some point, this ought to become something like

```suggestion
The Working Group has started documenting ethical issues associated with using Machine Learning on the Web, to help identify what mitigations its normative specifications should take into account. See [[ethical-webmachinelearning]].
```

But that'll have to wait for your draft to get into https://github.com/w3c/browser-specs or for you to add a [custom biblio entry](https://tabatkins.github.io/bikeshed/#biblio-custom) for it, and it'd be reasonable to wait for the WG to publish it as a [Draft Note](https://www.w3.org/2021/Process-20211102/#draft-note) before doing that. What you have seems fine for now. Thanks!

**Review by jyasskin — APPROVED** [review]

**Review by huningxin — APPROVED** [review]

File: index.bs (hunk 1 above)
Comment (anssiko) [reply index.bs:397]: The repo or the doc name might still change, so I intentionally did not want to codify the names yet. We’ll convert this into a biblio reference when we’re more solid.

**Comment by anssiko** [comment]
@dontcallmedom PTAL https://github.com/webmachinelearning/webnn/actions/runs/1451791945

**Comment by dontcallmedom** [comment]
this was a temporary issue - re-running the workflow fixed it

**Comment by anssiko** [comment]
Thanks for looking into this @dontcallmedom, however it seems https://www.w3.org/TR/webnn/ is not updated yet.

Specberus seems to think there's no homepage link and fails, while that's not true?

https://github.com/webmachinelearning/webnn/runs/4208398017
https://labs.w3.org/echidna/api/status?id=ac62262d-85d0-4dc7-8104-4251dad3904a
https://github.com/webmachinelearning/webnn/blob/gh-pages/index.html#L768

Could it be the training slash in the homepage link in the generated HTML? We haven't touched the boilerplate recently so this must be a tooling regression.

**Comment by dontcallmedom** [comment]
there was a pubrules update whose coordination with bikeshed was… [sub-optimal](https://lists.w3.org/Archives/Public/spec-prod/2021OctDec/0045.html). Our WebMLWG boilerplate in https://github.com/tabatkins/bikeshed-boilerplate probably needs an update - I won't be able to look into it myself today FWIW

**Comment by anssiko** [comment]
Thanks again @dontcallmedom.

I submitted https://github.com/tabatkins/bikeshed-boilerplate/pull/7 that I suspect might fix this.

I noticed @plehegar had updated bikeshed-boilerplate recently and is also aware of https://github.com/w3c/specberus/pull/1366 so looping him in here.

**Comment by anssiko** [comment]
I re-run the jobs after https://github.com/tabatkins/bikeshed-boilerplate/pull/7 was applied and that fixed the Specberus issue.

https://www.w3.org/TR/webnn/ is updated again.

**Pull request merged**